README.md

```diff
@@ -20,6 +20,7 @@ link = "https://github.com/me/my_plugin"
 
 # Optional image to showcase your asset.
 # Should be a png/jpg/gif/webp located next to your toml file, less than 2 MB in size.
+# We recommend a size of 551x310 pixels. (16:9 aspect ratio)
```

Comment:
I'm not seeing those values. I think it would be simpler and more future proof to ask for a 16:9 picture with a maximum width of 600px

Reply:
Works for me.